encrypted-forms.rst

```diff
@@ -0,0 +1,251 @@
+  
+These requirements are covered in more detail below.
+
+.. _security-concerns:
+  
+Security Concerns
+====================
+While ODK Collect attempts to remove all unencrypted copies of a finalized form and its attachments from the device, because ODK Collect uses third party applications for image capture, etc., and because of the potential for Forced Close events during the clean-up process, we cannot guarantee that all copies will have been destroyed. Furthermore, because of the way an SD card writes and deletes information, there is a possibility of this data being recoverable from the free space on the SD card. Your organization should investigate the extra steps needed to ensure all data is deleted from the SD cards on your ODK Collect devices and should establish procedures to periodically wipe and reinstall those devices.
+
+.. note::
+
+  Encrypting a form ensures that the finalized form is not readable and is not tampered with. However, there is nothing preventing a malicious adversary from the wholesale replacement of a finalized form with falsified data or the synthesis and submission of extra data — these are not contingencies that encrypted forms seek to address.
+  
+.. _config-briefcase:
+
+Configuration 
+====================
+
+For ODK Briefcase, you must download and install the *Java Cryptography Extension (JCE) Unlimited Strength Jurisdiction Policy Files 6* from the `Java download site <http://www.oracle.com/technetwork/java/javase/downloads/index.html#other>`_ ; for previous version of Java, it is available `here <http://www.oracle.com/technetwork/java/javase/downloads/jce-7-download-432124.html>`_ . This is required for decryption to be successful.
+
+To install the JCE:
+
+   1. Select the downloaded UCE zip archive file in the file viewer.
+   2. Right-click, choose :guilabel:`Extract files...` and extract the files from the downloaded zip archive.
+   3. Navigate into the extracted directory tree and copy the :file:`local_policy.jar` and :file:`US_export_policy.jar` files to the :file:`lib\security` directory under the installation directory of the Java Runtime Enviornment (JRE) of your computer, replacing earlier versions of these files.
+
+Windows
+~~~~~~~~~~~~~~~
+
+The JRE is usually installed here:
+
+:file:`C:\\Program Files\\Java\\jre1.x.x_xxx\\lib\\security`
+
+You might also have a JDK. If you do, you must also install the JCE files there:
+
+:file:`C:\\Program Files\\Java\\jdk1.x.x_xxx\\jre\\lib\\security`
+
+macOS
+~~~~~~~~~~~~~~~
+
+The JRE is usually found here:
+
+:file:`/Library/Internet Plug-Ins/JavaAppletPlugin.plugin/Contents/Home/lib/security`
+
+You might also have a JDK. If you do, you must also install the JCE files there:
+
+:file:`/Library/Java/JavaVirtualMachines/jdk1.x.x_xxx.jdk/Contents/Home/jre/lib/security`
+
+.. _upload-final-forms:
+
+Uploading Finalized Forms
+===========================
+
+.. code-block:: xml
+
+   <h:html xmlns=<http://www.w3.org/2002/xforms>
```

Comment:
<h:html xmlns="http://www.w3.org/2002/xforms"
        xmlns:h="http://www.w3.org/1999/xhtml"
	xmlns:orx="http://openrosa.org/xforms/">
@Divya063  Remove the three lines and replace them with this. This will fix your error.